governance.md

```diff
@@ -1,133 +1,229 @@
+* Have read the [Contributor Guide](https://spinnaker.io/community/contributing/).
+* Have made multiple contributions to the project or community. Contributions may include, but is not limited to:
+  * Authoring or reviewing PRs on GitHub
+  * Filing or commenting on Issues in GitHub
+  * Contributing to SIG or community discussions (e.g. meetings, Slack, email discussions)
+* Subscribed to [spinnaker-dev@googlegroups.com](TODO)
+* Sponsored by 2 Reviewers. **Note the following requirements for sponsors**:
+  * Sponsors must have close interactions with the prospective member - e.g. code/design/proposal review, coordinating issues, etc.
+  * Sponsors must be Reviewers or Approvers in the [Spinnaker org](https://github.com/spinnaker).
+  * Sponsors must be from multiple member companies to demonstrate integration across the community.
+* **[Open an issue](https://github.com/spinnaker/governance/issues/new) against the spinnaker/governance repo**
+  * Ensure your sponsors are @mentioned on the issue.
+  * Complete every item on the checklist.
+  * Make sure that the list of contributions included are representative of your work on the project.
+* Have your sponsoring reviewers reply confirmation of sponsorship: `:+1:`.
+* Once your sponsors have responded, your request will be reviewed by the TOC. Any missing information will be requested.
+
+#### Responsibilities and Privileges
 
-Duties
+* Responsive to issues and PRs assigned to them.
+* Responsive to mentions of SIG teams they are members of.
+* Active owner of code they have contributed (unless ownership is explicitly transferred).
+  * Code is well tested.
+  * Tests consistently pass.
+  * Addresses bugs or issues discovered after code is accepted.
+* They can be assigned to issues and PRs, and people can ask members for reviews.
 
-* Review, provide meaningful feedback and LGTM PRs
-* Triage GitHub issues, community forum submissions
-* Provide general support in Slack
+**Note**: Members who frequently contribute code are expected to proactively perform code reviews and work towards becoming a Reviewer.
 
-Qualification guidelines
 
-* Known active participant in the community
-* Has submitted at least 5 PRs of significant scope that are correspondingly merged
-* Sponsored by 2 Approvers
+### Reviewer
 
-[Current Reviewers](https://github.com/spinnaker/governance/blob/master/membership.yml)
+Reviewers are able to review code for quality and correctness on some part of the project. They are knowledgable about both the codebase and software engineering principles. Reviewership is required before becoming an Approver (except under specific conditions; see Approver).
 
-If you’d like to nominate yourself as a Reviewer, please create an [Issue in the governance repository](https://github.com/spinnaker/governance/issues). 
+Reviewer status is scoped to a part of the codebase.
```

Comment:
Did we end up deciding on repo-level status?  Or is the plan that for now it will apply to the whole codebase but we might split out parts as needed (ex: kleat/spin/halyard)?